src/base/RecoveryCodeCollection.php

```diff
@@ -0,0 +1,128 @@
+<?php
+declare(strict_types=1);
+
+namespace hiqdev\yii2\mfa\base;
+
+use Yii;
+use yii\base\BaseObject;
+
+class RecoveryCodeCollection extends BaseObject
+{
+    protected array $codes = [];
```

Comment:
there are no property-type-hints in php 7.0 :(

Reply:
:-( changed locally, push later